Tests.Integration.Presentation.Web/ItSystem/V2/ItSystemsApiV2Test.cs

```diff
@@ -590,6 +590,46 @@ public async Task Can_Create_Update_And_Delete_ExternalReference()
             Assert.Empty(afterDelete.ExternalReferences);
         }
 
+        [Fact]
+        public async Task Can_Get_ItSystem_Permissions()
+        {
+            //Arrange
+            var orgId = TestEnvironment.DefaultOrganizationId;
+            var token = (await HttpApi.GetTokenAsync(OrganizationRole.GlobalAdmin)).Token;
+
+            var system = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), orgId, AccessModifier.Public);
+            
+            //Act
+            var permissionsResponseDto = await ItSystemV2Helper.GetPermissionsAsync(token, system.Uuid);
+
+            //Assert
+            var expected = new ResourcePermissionsResponseDTO
+            {
+                Read = true,
```

Comment:
you need another test to verify that regular non-admin users have different permissions... in that way we see that it actually maps the results from the response